maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -105,7 +105,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 
 	private void testing(Run run) throws Exception {
 		try {
-			System.out.println("Test " + run);
+			System.out.println(Strings.join("\n", new Object[] {
```

Comment:
the format string would be super long and I wasn't sure how to deal with that without also having `+`s all over the place.
the varargs join doesn't work when the delimiter is the same type as the args you mean to delimit.